tests/unit/models/GenericComponentDefinition.spec.js

```diff
@@ -0,0 +1,25 @@
+import GenericComponentDefinition from 'src/models/GenericComponentDefinition';
+
+describe('Test class: GenericComponentDefinition', () => {
+  describe('Test constructor', () => {
+    it('Check variable instantiation', () => {
+      const component = new GenericComponentDefinition();
+
+      expect(component.available).toEqual(false);
+    });
+
+    it('Check passing undefined variables to constructor', () => {
```

Comment:
not an `undefined variables`, it's an empty object (so it's defined).

Reply:
done